### ISSUE TITLE
Operation semantics of __bf16 datatype

### ISSUE THREAD

**Issue by joshua-arch1** [riscv-non-isa/riscv-elf-psabi-doc#384]
I missed yesterday's meeting. I checked the slides and found the __bf16 issue was discussed. Were operation semantics discussed in the meeting? Is there a plan how we will decide these semantics later?

**Comment by kito-cheng** [comment]
Short answer: Yes, but we don't decide semantic yet.

However I am not sure psABI is the best place to discuss or not since we don't have too much floating point expert here, so https://github.com/riscv/riscv-bfloat16 should be better IMO.

c.c. @nick-knight is the only floating point expert I kown :P

**Comment by nick-knight** [comment]
I agree with Kito that the RISC-V bfloat16 task group may be a better venue.
